### DISCUSSION TITLE
Running scripts to fetch data from an external API?

### DISCUSSION THREAD

**Discussion by stanifert** [supabase/supabase#3190] [Questions] [answered]
Hello there! 

Apologies if this is a very/too basic question, but is it possible to run scripts on a Supabase instance? For example, have the server fetch data from an API every X time interval. I'm exploring creating my own database that collects data from multiple sources and stores it. 

If this is a very basic thing, i.e. something that's very easy to do with Postgres, could someone point me in the right direction for info/guides on how to achieve this? I'm an iOS developer with little web/server experience and looking to learn.

Thank you!

**Comment by prescience-data** [comment, marked as answer]
Two extensions that you can enable in Supabase dashboard that will achieve this:
CRON: https://github.com/citusdata/pg_cron
HTTP: https://github.com/pramsey/pgsql-http
👍

**Reply by stanifert** [reply]
Awesome :) Have never heard of those, now my google powers are extended. Thank you!
